apps/system/js/input_window_manager.js

```diff
@@ -422,6 +422,8 @@
     this._currentWindow = nextWindow;
   };
 
+  InputWindowManager.prototype.HIDE_INPUT_WINDOW_TIMEOUT = 200;
```

Comment:
Quick FYI: the old timeout we used at KeyboardManager for this matter was 100. See: https://github.com/mozilla-b2g/gaia/commit/35fd27531d52a371835f5188a2a2cef446c299e7